exchanges/bybit/convert_types.go

```diff
@@ -0,0 +1,114 @@
+package bybit
+
+import (
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/encoding/json"
+	"github.com/thrasher-corp/gocryptotrader/types"
+)
+
+// Consts for WalletAccountType
+const (
+	Funding  WalletAccountType = "eb_convert_funding"
+	Uta      WalletAccountType = "eb_convert_uta"
+	Spot     WalletAccountType = "eb_convert_spot"
+	Contract WalletAccountType = "eb_convert_contract"
+	Inverse  WalletAccountType = "eb_convert_inverse"
+)
+
+// WalletAccountType represents the different types of wallet accounts
+type WalletAccountType string
+
+// CoinResponse represents a coin that can be converted
+type CoinResponse struct {
```

Comment:
🚧 I think if this is pidgeonholed to just conversion, then the name lacks context.
```suggestion
// ConvertCoinResponse represents a coin that can be converted
type ConvertCoinResponse struct {
```